prime-router/settings/organizations-prod.yml

```diff
@@ -156,7 +156,9 @@
     - name: elr
       organizationName: nd-doh
       topic: covid-19
-      jurisdictionalFilter: [ "matches(ordering_facility_state, ND)" ]
+      jurisdictionalFilter:
+        - matches(ordering_facility_state, ND)
+      deidentify: false
```

Comment:
To be safe, I think we should change the default for this to be deidentify: true   if not specificied.